src/fs/serve_file.rs

```diff
@@ -0,0 +1,84 @@
+use crate::log;
+use crate::{Body, Endpoint, Request, Response, Result, StatusCode};
+use std::io;
+use std::path::Path;
+
+use async_std::path::PathBuf as AsyncPathBuf;
+use async_trait::async_trait;
+
+pub struct ServeFile {
+    file: AsyncPathBuf,
+}
+
+impl ServeFile {
+    /// Create a new instance of `ServeFile`.
+    pub(crate) fn init(file: impl AsRef<Path>) -> io::Result<Self> {
```

Comment:
```suggestion
    pub(crate) fn init(path: impl AsRef<Path>) -> io::Result<Self> {
```

Reply:
Thank you! Done, also in a couple of other places.